src/js/popup.js

```diff
@@ -245,6 +245,15 @@ function limitAccess (callback = false) {
 }
 
 $(document).ready(function () {
+    // set the display:block of the content in a timeout to avoid resizing of popup
+    setTimeout(() => {
```

Comment:
While there is nothing wrong about using es6 style `() => {}` functions, we haven't used them anywhere else in the extension. So many for consistency we stick with `function () {}` until we have time to go through and update the entire extension.

Reply:
Okay makes sense, I will update it to be consistent with the rest of the code :) Thanks 💯